product_docs/docs/migration_portal/4.0/04_mp_migrating_database/02_mp_schema_assessment.mdx

```diff
@@ -85,3 +85,41 @@ You can optionally select **Generate PDF** to save the report in PDF format. You
     Migration Portal doesn't assess sensitive PL/SQL source code hidden in Oracle wrapped objects. These wrapped objects aren't included in the assessed objects count, and therefore the true value of compatibility percentage might be different from the value calculated in the assessment report.  
     
 See the note in [Performing a schema extraction](01_mp_schema_extraction#wrapped-objects) for more information about wrapped objects.
+
+## Evaluate an assessment report
+
+The assessment report presents data and information about the compatibility of objects and features in your schemas and the overall project. 
+
+- Object: A CREATE or ALTER TABLE  statement that adds a constraint in the uploaded DDL.
+- Feature: An Oracle clause or a construct that is used like an object. A feature can be used multiple times in one or more objects.
+
+### Object compatibility
+
+An object is compatible if it successfully executes on a target database. In the assessment report, you see a summary of feature occurrences based on the following levels of compatibility:
```

Comment:
The phrase "summary of feature occurrences" is not correct in this context.  It would be better to say something like "you see a summary of object compatibility based on the following categories:".